src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -46,16 +46,29 @@ public bool TryRegister(SafeCloseSocket socket, Interop.Sys.SocketEvents current
             }
         }
 
-        private const int EventBufferCount = 64;
+        private const int EventBufferCount =
```

Comment:
It applies:
MaxHandles for DEBUG is calculated as: `private static readonly IntPtr MaxHandles = (IntPtr)(EventBufferCount * 2);`
I will halve the value because of the 2 engines.